src/app/main-api-handler.ts

```diff
@@ -48,7 +48,7 @@ import {
 let swiftSearchInstance;
 try {
   // tslint:disable-next-line:no-var-requires
-  const { SSAPIBridge } = require('swift-search');
+  const { SSAPIBridge } = {} as any; // require('swift-search');
```

Comment:
[Question]: swift-search is not compatible with the upgrade to Electron 19 / Node 16?

Reply:
Yes, there are other major API changes that needs to be made in swift search module